single-campaign.php

```diff
@@ -86,6 +86,7 @@ static function ($page) {
 $context['social_accounts'] = $post->get_social_accounts($context['footer_social_menu'] ?: []);
 $context['page_category'] = $data_layer['page_category'];
 $context['post_tags'] = implode(', ', $post->tags());
```

Comment:
Just a note, we don't have categories and tags for campaign post type.

Reply:
Ah, good catch! I'll remove both then 👍